src/Symfony/Component/Lock/Store/RedisStore.php

```diff
@@ -33,13 +33,13 @@ class RedisStore implements StoreInterface
     private $initialTtl;
 
     /**
-     * @param \Redis|\RedisArray|\RedisCluster|\Predis\ClientInterface $redisClient
-     * @param float                                                    $initialTtl  the expiration delay of locks in seconds
+     * @param \Redis|\RedisArray|\RedisCluster|RedisProxy|RedisClusterProxy\Predis\ClientInterface $redisClient
```

Comment:
@jderusse There is a bug: no pipe in `RedisClusterProxy\Predis\ClientInterface` leads to wrong class FQN.